modules/analysis/src/service/mod.rs

```diff
@@ -676,6 +676,10 @@ impl AnalysisService {
                         context.put_value("cpe", Value::from(&package.cpe));
                         context.put_value("purl", Value::from(&package.purl));
                         context.put_array("purl", purls);
+                        context.put_array(
```

Comment:
It may be worth checking the DB implementation of the analysis endpoint for further inconsistencies. It may be that `part` and `purl` aren't the only one.